tests/operator_test.go

```diff
@@ -39,15 +39,37 @@ var _ = Describe("Operator tests", func() {
 		Expect(r.Spec.TLS.Termination).To(Equal(routev1.TLSTerminationReencrypt))
 	})
 
-	It("add cdi-sa to anyuid scc", func() {
-		if !controller.IsOpenshift(f.K8sClient) {
+	It("should create a prometheus service in cdi namespace", func() {
```

Comment:
I think this shouldn't be here, its also causing failures when compiling.

Reply:
ugh, yeah this was not a clean merge.  Updated.